README.md

```diff
@@ -161,6 +161,18 @@ To install zlib-ng system-wide using the configure script:
 make install
 ```
 
+Alternatively, you can build and install zlib-ng using [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager:
```

Comment:
We should have a header `### vcpkg` above this. So we would have CMake, configure, and vcpkg headings.

Reply:
Add "the" after "using"

Reply:
Adding `vcpkg` head will be more intuitive.